odoo/addons/base/tests/test_translate.py

```diff
@@ -687,23 +687,31 @@ def _test_create_empty(self, empty_value):
         self.assertEqual(group.with_context(lang='fr_FR').comment, 'French comment 2')
         self.assertEqual(group.with_context(lang='en_US').comment, 'French comment')
 
-    def test_field_selection(self):
-        """ Test translations of field selections. """
+    def test_update_field_translations(self):
```

Comment:
Why did you remove the method `test_field_selection` ?
Please put it back (unless there is a good reason.)